test/clj_pgp/test/encryption.clj

```diff
@@ -105,6 +106,45 @@
               (is @error-occured? "A PGP error was simulated but not passed to the error handler."))))))))
 
 
+(def ^:private packet-header-byte
+  "
+  From right to left:
+    - Bits 1–0 are set to `00` to indicate a one-octet length packet.
+    - Bits 5–2 specify the packet's tag, which is `0001`.
+    - Bit 6 is `0` to indicate an 'old format' packet.
+    - Bit 7 is always `1` in a packet header.
+  See RFC 4880 section 4.2.
+  "
+  2r10000100)
+
+
+(def ^:private packet-length-byte
+  "
+  A packet's length doesn't include the first two header bytes.
+  This is an empty packet, so we're just using `0`
+  "
+  2r00000000)
+
+
+(def ^:private packet-count-byte
+  "
+  I honestly don't understand why we need this byte or what it represents
+  but bouncycastle's constructor will throw without it.
+  Arrived at it via LLM + trial + error
```

Comment:
What would a future reader of the test code here do with this information?

Reply:
Not super important.  If, for example, someone who actually knows what they're doing sees this test and wonders why the final byte is `2r10000000` instead of `2r11000000`, I want to answer their "Chesterton's Fence" question.

That being said, I was very close to just inlining all of the bytes and hand-waving it as "it works".  The private vars were broken out just to give the individual bytes some documentation.

Would it be better to just do this?
```clojure
(deftest empty-packet-handling
  (let [packet-header-byte 2r10000100
        packet-length-byte 0
        packet-count-byte 2r10000000
        empty-packet (PGPEncryptedDataList.
                       (byte-array [packet-header-byte
                                    packet-length-byte
                                    packet-count-byte]))
        accumulator [:accumulated :data]
        rf (fn [_acc {_data :data}]
             (is false "reducing function should not be called"))]
    (is (= accumulator (pgp-msg/reduce-message empty-packet {} rf accumulator))
        "When given an empty packet, we should return the accumulator instead of throwing")))
```

Reply:
I went ahead and pushed that as the change.  Just let me know if any of that extra context is worth adding as code comments.

Reply:
I meant specifically the "Arrived at it via LLM + trial + error" part, the rest of the comments were fine!